final_project/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+'''class Meta- nested class, tells Django this is not a real class and to not bring it in as a table,
+therefore it's an abstract class'''
```

Comment:
`abstract = True` property is the reason why this class/model is abstract, not the `Meta` class.